### PR TITLE
Workspace: Layer panel bg focus color

### PR DESCRIPTION
**Pull request by dmmulroy**
## Summary

This fixes a bug where the focused/selected layers background colors was an off blue instead of gray.
## Relevant Technical Choices

- Added `theme.colors.bg.v17` as `'#5e615c'`

## User-facing changes

The background color should now be gray

## Testing Instructions

Before
before

After
after

---

 #4060

### CODE REVIEW

**Review by BrittanyIRL — APPROVED** [review]
LGTM, color is updated to gray. 🚀

File: assets/src/edit-story/theme.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ const theme = {
       v14: '#3E445B',
       v15: '#212433',
       v16: '#2C2C2C',
+      v17: '#5e615c',
```
Comment (barklund) [inline assets/src/edit-story/theme.js]: As this is known as `gray24` in the color scheme, can we call it that here too (like [the other named colors](https://github.com/google/web-stories-wp/pull/3787/files#diff-e9f19c950bf7580cdde6db2b8ffb5ea5R119-R120))?

Also, it's technically a foreground color (as per the design guide), so it should be moved to the `fg` section.

Screen Shot 2020-08-20 at 19 34 31

File: assets/src/edit-story/components/panels/layer/layer.js [hunk 2]
```diff
@@ -52,9 +52,9 @@ const LayerButton = styled.button.attrs({
 
   ${({ isSelected, theme }) =>
     isSelected &&
-    `
-    background: ${rgba(theme.colors.accent.primary, 0.14)};
-  `}
+    css`
+      background: ${rgba(theme.colors.bg.v17, 1)};
```
Comment (barklund) [inline assets/src/edit-story/components/panels/layer/layer.js]: `rgba()` is only needed, when alpha is less than 1. For this, please just use the color directly:

```js
      background: ${theme.colors.fg.gray24};
```

**Review by barklund — DISMISSED** [review]
A few comments

**Comment by swissspidy** [comment]
This apparently overlapped with some of the work in #3787, so there's now a merge conflict.

File: assets/src/edit-story/components/panels/layer/layer.js (hunk 2 above)
Comment (dmmulroy) [reply assets/src/edit-story/components/panels/layer/layer.js]: fixed

Comment (dmmulroy) [reply assets/src/edit-story/components/panels/layer/layer.js]: fixed

File: assets/src/edit-story/theme.js (hunk 1 above)
Comment (dmmulroy) [reply assets/src/edit-story/theme.js]: done

**Review by barklund — APPROVED** [review]

**Pull request merged**